src/actions/authEvents.actions.js

```diff
@@ -17,6 +17,34 @@ export function createEventSuccess(payload) {
 	};
 }
 
+export function editEventSuccess(payload) {
+	return {
+		type: types.EDIT_EVENT_SUCCESS,
+		payload
+	};
+}
+
+export function deleteEvent(payload) {
+	return {
+		type: types.DELETE_EVENT_SUCCESS,
+		payload
+	};
+}
+
+export function getEventCategory(payload) {
+	return {
+		type: types.GET_EVENT_CATEGORY,
+		payload
+	};
+}
+
+export function rsvpEventSuccess(payload) {
+	return {
+		type: types.SUCCESSFUL_RSVP,
+		payload
+	};
+}
+
 export function userEventsAction() {
```

Comment:
wouldn't it be better if you renamed this action in a way that explains what it does? so something like `getUserEventsAction`